apis/r/R/utils.R

```diff
@@ -110,6 +110,65 @@ uns_hint <- function(type = c('1d', '2d')) {
   }
 }
 
+.type_hint <- function(type) {
```

Comment:
I know this is internal & doesn't result in any `.Rd` content but it's a a non-trivial function with zero comments

please add at least some doc couments at the top: why is this function here? what is its nominal use? what are one or two sample inputs & outputs?

Reply:
Added in 568dd6b